index.html

```diff
@@ -370,5 +373,66 @@ <h3>LET’S CONNECT</h3>
       </section>
     </main>
     <!-- end sectionsix -->
+    <!-- start footer -->
+    <div class="white">
+      <footer class="footer" id="footer">
+        <div class="container">
+          <div class="footer__container__parfoot">
+            <h3>Contact me</h3>
+            <p>
+              If you have an application you are interested in developing, a
+              feature that you need built or a project that needs coding. I’d
+              love to help with it
+            </p>
+            <form action="https://formspree.io/f/mvoyalje" method="post">
+              <div class="form__control">
+                <input
+                  type="text"
+                  placeholder="Name"
+                  class="name"
+                  id="name"
+                  maxlength="30"
+                  required
+                  name="name"
+                />
+              </div>
+              <div class="form__control">
+                <input
+                  type="email"
+                  placeholder="name@example.com"
+                  class="email"
+                  id="email"
+                  name="email"
+                  required
+                />
+              </div>
+              <div class="form__control">
+                <textarea
+                  id=""
+                  cols="30"
+                  rows="10"
+                  class="textarea"
+                  id="textarea"
+                  placeholder="Write your message here"
+                  maxlength="500"
+                  name="textarea"
+                  required
+                ></textarea>
+              </div>
+              <div class="form__control">
+                <input
+                  type="submit"
+                  placeholder=""
+                  class="submit"
+                  id="submit"
+                  value="Get in touch"
+                />
+              </div>
+            </form>
+          </div>
+        </div>
+      </footer>
```

Comment:
**[OPTIONAL]** - For semantic purposes and accessibility, I'd suggest that you don't wrap your contact section in a `footer` tag. You could use a `section` tag for this.

Reply:
Thank you ☺️ I appreciate your review you are an amazing reviewer again thank you for your effort 🎉.